tests/test_scrape_mensa.py

```diff
@@ -1,12 +1,47 @@
-import pytest
+from textwrap import dedent
+from typing import  List, Tuple
+import datetime
+
 import requests
+import pytest
 
-from lunchbot.scrape_mensa import parse_menu
+from lunchbot.scrape_mensa import parse_menu, VEGY_TYPES
 from lunchbot.scrape_mensa import read_page
-from lunchbot.scrape_mensa import MENU_PARAMS
 from lunchbot.scrape_mensa import URI
 
 
+def generate_test_html(
+    days: List[datetime.date],
+    items: List[Tuple[str, str]],
+    day_index: int = 0,
+):
+    """Encapsulate all html generation logic, such that if the website changes, you
+    only need to change this function.
+    """
```

Comment:
Nice! Do you think, it is worth it adding some real HTML as soon as the web page provides some? How would we do it? Replacing this function or adding a second one?

Reply:
We could add a "real life" test with the actual html as an independent test.

I guess we don't want to generate all test cases with the actual html, then it might be difficult to debug.

Reply:
Hm yes, makes sense.